### PR TITLE
feat: customize aside html element

### PR DESCRIPTION
**Pull request by MekalaNagarajan-Centrica**

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import { defineConfig } from "astro/config";
 import starlight from "@astrojs/starlight";
 import lit from "@astrojs/lit";
+import { nucleusRemarkAsides } from "./src/plugins/nucleus-aside";
 
 export default defineConfig({
+  markdown: {
+    remarkPlugins: [nucleusRemarkAsides()]
```
Comment (andij) [inline astro.config.mjs:8]: Suggestion to change this:

```suggestion
import { nucleusAside } from "./src/plugins/nucleus-aside";

export default defineConfig({
  markdown: {
    remarkPlugins: [nucleusAside()]
```

**Review by andij — APPROVED** [review]

File: astro.config.mjs (hunk 1 above)
Comment (MekalaNagarajan-Centrica) [reply astro.config.mjs:8]: It is a practice for all remark plugins to include `remark` keyword in the plugin name.

Comment (andij) [reply astro.config.mjs:8]: Okay, no probs. Could we instead change the name of the file to be `nucleus-remark-aside` ?

Comment (MekalaNagarajan-Centrica) [reply astro.config.mjs]: done

**Review by andij — APPROVED** [review]

**Pull request merged**